apps/storefront-e2e/src/support/page-objects/cart.page.ts

```diff
@@ -8,11 +8,11 @@ export class CartPage extends AbstractSFPage {
   private cartTotals = new CartTotalsFragment();
 
   waitForLoaded(): void {
-    this.getEmptyCartMessage().should('be.visible');
+    this.getCartEntriesWrapper().should('exist');
   }
 
   getCartEntriesWrapper = () => cy.get('oryx-cart-entries');
-  getEmptyCartMessage = () => cy.get('oryx-content-text');
+  getEmptyCartMessage = () => cy.contains('Your shopping cart is empty');
```

Comment:
It's not get then.

Reply:
`contains` is a getter - https://docs.cypress.io/api/commands/contains

Reply:
> Get the DOM element containing the text. DOM elements can contain more than the desired text and still match. Additionally, Cypress [prefers some DOM elements](https://docs.cypress.io/api/commands/contains#Notes) over the deepest element found.

Reply:
Ah, I see. But it's better to not rely on the text...